docs/docs/guides/how-to-promote-content.md

```diff
@@ -12,17 +12,17 @@ Promoting content enables you to copy content from one project to another. It's
 
 ## How promoting works
 
-If you are promoting a chart, from one `development project` to a `production project`, this will happen:
+If you are promoting a chart, from a `development` project to a `production` project, this will happen:
 
-- If the chart exists in both the `development project` and the `production project`, the chart in the `production` will be updated with the changes from `development`. You can always revert this chart to a previous version using [version history](./version-history)
-- If the chart is new, we will replicate this chart into the `production project`. We will also create a new space if needed.
+- If the chart exists (with the same name) in both the `development` project and the `production` project, the chart in `production` will be updated with the changes from `development`. You can always revert this chart to a previous version using [version history](./version-history)
```

Comment:
`(with the same name)` This is not 100% right, but close. 

You can rename a chart, and still keep promoting the same chart, becuase we use `slugs` to identify chats internally. Slugs are initially created based on the chart/dashboard name, and never modified. 

I don't think there is a better way to explain what a slug is to a user, ,so we can keep it as it is.